WRIO-InternetOS/start.js

```diff
@@ -0,0 +1,187 @@
+var cssUrl = require('./js/global').cssUrl,
```

Comment:
@AlexeyAnshakov no, it's another start.js, in WRIO-InternetOS folder, it not exposed to html file and will be most likely confused with final start.js